go.sum

```diff
@@ -139,6 +137,9 @@ github.com/onsi/ginkgo v1.7.0/go.mod h1:lLunBs/Ym6LB5Z9jYTR76FiuTmxDTDusOGeTQH+W
 github.com/onsi/gomega v1.4.1/go.mod h1:C1qb7wdrVGGVU+Z6iS04AVkA3Q65CEZX59MT0QO5uiA=
 github.com/onsi/gomega v1.4.3 h1:RE1xgDvH7imwFD45h+u2SgIfERHlS2yNG4DObb5BSKU=
 github.com/onsi/gomega v1.4.3/go.mod h1:ex+gbHU/CVuBBDIJjb2X0qEXbFg53c61hWP/1CpauHY=
+go.etcd.io/bbolt v1.3.3/go.mod h1:IbVyRI1SCnLcuJnV2u8VeU0CEYM7e686BmAb1XKL+uU=
```

Comment:
Shouldn't this have eliminated v1.3.3?

Reply:
Oh, it's because of

github.com/decred/dcrwallet/wallet/v3@v3.1.1-0.20191230143837-6a86dc4676f0 go.etcd.io/bbolt@v1.3.3

And github.com/decred/dcrwallet/wallet[/v3] is no longer a module, so we need to update to `decred.org/dcrwallet.

Reply:
So updating the direct bbolt depend to 1.3.4 forces it to be used for wallet/v3, so this is good.  We'll update to `decred.org/dcrwallet` in another PR.